modules/volume/rendering/basicvolumeraycaster.h

```diff
@@ -76,7 +76,7 @@ class BasicVolumeRaycaster : public VolumeRaycaster {
 
     void setVolumeTexture(std::shared_ptr<ghoul::opengl::Texture> texture);
     std::shared_ptr<ghoul::opengl::Texture> volumeTexture() const;
-    void setTransferFunction(std::shared_ptr<TransferFunction> transferFunction);
+    void setTransferFunction(std::shared_ptr<openspace::TransferFunction> transferFunction);
```

Comment:
No need to explicitly specify the `openspace` namespace qualifier here as we are already in it